scripts/apps/workspace/content/controllers/ContentProfilesController.ts

```diff
@@ -158,7 +159,8 @@ export function ContentProfilesController($scope, $location, notify, content, mo
 
         content.updateProfile(e.original, diff)
             .then(refreshList.bind(this, false), reportError)
-            .then(this.toggleEdit.bind(this, null));
+            .then(this.toggleEdit.bind(this, null))
+            .then(this.savingInProgress = false);
```

Comment:
A function should be passed to `.then`, not expression. It would be `() => {this.savingInProgress = false}` in this case.